src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -167,50 +153,49 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
         if (distributeToRecipients) {
             String optionFragment =
                     Templates.populateTemplate(optionFragmentTemplate,
-                            "${qnIdx}", Integer.toString(qnIdx),
-                            "${responseIdx}", Integer.toString(responseIdx),
-                            "${optionIdx}", "0",
-                            "${disabled}", sessionIsOpen ? "" : "disabled",
-                            "${constSumOptionVisibility}", "style=\"display:none\"",
-                            "${constSumOptionPoint}", existingConstSumResponse.getAnswerString(),
-                            "${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}", Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
-                            "${constSumOptionValue}", "");
+                            Templates.FeedbackQuestionDetailsConsts.QUESTION_INDEX, Integer.toString(qnIdx),
```

Comment:
Importing that static class like `import static teammates.common.util.Templates.FeedbackQuestionDetailsSlots.*` and using the Const fields directly will be good ?